### ISSUE TITLE
Catcher plate shouldn't be mirrored when catcher direction changes

### ISSUE THREAD

**Issue by ekrctb** [ppy/osu#11112]
**Describe the bug:** 

Catcher sprite is mirrored as the catcher direction changes, but the caught objects on the plate and hit lighting shouldn't change relative position.
Stable does not mirror catcher plate.

**Comment by peppy** [comment]
Yeah, not sure about this one. I kinda like it the new way, but i think some players will complain about this change.

At very least, we want it to be customisable and use the old display method for legacy skins.

**Comment by amberkowalski** [comment]
> At very least, we want it to be customisable and use the old display method for legacy skins.

Is this still the current accepted solution? If so I'd be happy to implement it.

**Comment by peppy** [comment]
I think we definitely want to leave this as a potential customisation for the future (ie. the skin or user can choose whether they want this to be on or off), so that sounds good.
